lib/convert.ts

```diff
@@ -159,7 +159,7 @@ export default class Convert {
       case 'deleted':
         return [{uri: Convert.pathToUri(fileEvent.path), type: ls.FileChangeType.Deleted}];
       case 'renamed': {
-        const results = [];
+        const results: Array<{ uri: string, type: ls.FileChangeType }> = [];
```

Comment:
TypeScript uses a type of `never[]` when you define a plain array with no typing.  You have to make the type specific so that the type system doesn't give a bunch of weird errors.